src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj

```diff
@@ -9,6 +9,13 @@
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="2.0.6" />
```

Comment:
It look like all the json recordings have `Microsoft.Azure.Management.ResourceManager.ResourceManagementClient/1.6.0.0`